mobileapp/.gitignore

```diff
@@ -9,6 +9,7 @@ fastlane/screenshots
 fastlane/metadata
 fastlane/test_output
 fastlane/fastlane-key.json
+gc_keys.json
```

Comment:
Can't remember if this one is actually needed or not, let's make sure you try without it next time we make a mobile release. If it is, I can see if I can do something to make it shared/accessible :)